aspnetcore/security/authentication/azure-ad-b2c-webapi.md

```diff
@@ -170,12 +170,14 @@ To make an authenticated request to the web API, a bearer token is required. Pos
+   |       <strong>Auth URL</strong>        | `https://login.microsoftonline.com/<tenant domain name>/oauth2/v2.0/authorize?p=B2C_1_SiUpIn` |  Replace <em>&lt;tenant domain name&gt;</em> with the tenant's domain name. **IMPORTANT**: This URL must have the same domain name as what is found in `AzureAdB2C.Instance` in the web API's *appsettings.json* file. See NOTE&dagger;.            |
    |       <strong>Client ID</strong>       |                <em>&lt;enter the Postman app's <b>Application ID</b>&gt;</em>                 |                                                                                                                                                                                                                                                                              |
    |         <strong>Scope</strong>         |         `https://<tenant domain name>/<api>/user_impersonation openid offline_access`         | Replace <em>&lt;tenant domain name&gt;</em> with the tenant's domain name. Replace <em>&lt;api&gt;</em> with the App ID URI you gave the web API when you first registered it (in this case, `api`). The pattern for the URL is: <em>https://{tenant}.onmicrosoft.com/{api-id-uri}/{scope name}</em>. |
    |         <strong>State</strong>         |                                 <em>&lt;leave blank&gt;</em>                                  |                                                                                                                                                                                                                                                                              |
    | <strong>Client Authentication</strong> |                                Send client credentials in body                                |                                                                                                                                                                                                                                                                              |
 
+    > [!NOTE]
+    > &dagger; The **Auth URL** can be confusing. The policy settings dialog in the Azure Active Directory B2C portal displays two possible URLs, one in the format `https://login.microsoftonline.com/<tenant domain name>/<additional path information>`, and the other in the format `https://<tenant name>.b2clogin.com/<tenant domain name>/<additional path information>`. It's **critical** that the domain found in in `AzureAdB2C.Instance` in the web API's *appsettings.json* file matches the one used in the web app's *appsettings.json* file. This is the same domain used for the Auth URL field in Postman. Additionally, note that Visual Studio uses a slightly different URL format than what's displayed in the portal. As long as the domains match, the URL works.
```

Comment:
Remove the space between the dagger and the "T" ... just in case they allow this. lol

The structure of the second sentence is still a bit strange because it's a list of two items. Also note that recent discussion on localization might be leading to using braces over angle brackets for user-supplied segments and not code-fencing links for user-supplied segments. Localization should be able to localize the content if we don't code-fence links. I defer to @Rick-Anderson and @scottaddie for the latest word on that.

I guess (speculate) that it might go like this ...

> The policy settings dialog in the Azure Active Directory B2C portal displays two possible URLs: one in the format *https://login.microsoftonline.com/{tenant domain name}/{additional path information}\*, and the other in the format *https://{tenant name}.b2clogin.com/{tenant domain name}/{additional path information}*.

Reply:
We're still waiting on a final anser on this ... see: https://github.com/aspnet/Docs/pull/8597

If we don't code-fence, we get localization. We can provide user-supplied segments in braces (with spaces between words) and get localization. It might be *the future*.

Reply:
I went ahead and applied that style for links over the entire doc. I think it looks better now.